src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1629,6 +1639,14 @@ https://api.livechatinc.com/v3.4/agent/action/tag_thread \
 
 ### Untag Thread
 
+Untags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be untagged. 
+- One or more tags can be removed from the specific **thread**. 
+- Tags are case sensitive.
```

Comment:
Hmm, I'm thinking if this belongs here, as we know already that tags are case sensitive from the previous "Tag thread" description and it's more of a tag-specific thing rather than method-specific information 🤔

Reply:
I put that rather 'just to be sure', but now that I am thinking about what you have written, I am more for removing it from here.

Reply:
Removing it from this description would assume someone has written the Tag Thread method description. I they haven't and we remove the description, one might not one that calling Untag Thread with `support` and `Support` is removing two separate tags.

Reply:
Good point, so this should definitely stay.